src/pages/ctrader-manage/_ctrader-manage-data.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import AddAccountImage from 'images/common/ctrader/add-account.png'
+import AddMoreAccountsImage from 'images/common/ctrader/add-more-accounts.png'
+import AddAccountMobileImage from 'images/common/ctrader/add-account-mobile.png'
+import { TString } from 'types/generics'
+import { LinkUrlType } from 'features/types'
+
+type ParagraphType = {
+    id?: number
+    text: TString
+    components?: React.ReactElement[]
+}
+
+type CtraderManageDataType = { [page: string]: CtraderManageProps }
+
+export type CtraderManageProps = {
+    title: TString
+    paragraph: ParagraphType
+    steps?: ParagraphType[]
+    image_url?: string
+    image_url_mobile?: string
+    button_url: LinkUrlType
+    button_text: TString
+}
+
+export const ctrader_manage_data: CtraderManageDataType = {
+    add_account: {
+        title: '_t_Creating a live cTrader account_t_',
+        button_url: {
+            type: 'non-company',
+            target: '_blank',
+            href: 'https://app.deriv.com/appstore/traders-hub',
```

Comment:
I believe that it's company's one, but you need update `external atom`

Reply:
These links might change in future. Can we keep them hardcoded for now?